packages/augur-core/tests/trading/test_orderbook_ordering.py

```diff
@@ -1,28 +1,28 @@
 #!/usr/bin/env python
 
 from ethereum.tools import tester
-from utils import longTo32Bytes, longToHexString, bytesToHexString, fix, AssertLog, stringToBytes, EtherDelta, PrintGasUsed, BuyWithCash
+from utils import longTo32Bytes, longToHexString, bytesToHexString, fix, AssertLog, stringToBytes, EtherDelta, PrintGasUsed, BuyWithCash, nullAddress
 from constants import ASK, BID, YES, NO
 
 def test_correct_order_for_same_price(contractsFixture, cash, market):
     createOrder = contractsFixture.contracts['CreateOrder']
     orders = contractsFixture.contracts['Orders']
     tradeGroupID = longTo32Bytes(42)
-    nullAddress = longTo32Bytes(0)
+    nullOrder = longTo32Bytes(0)
```

Comment:
Nit: might want to define this in utils or up above since it's defined twice here.